pubpub/platform
# fix: properly pass superadmin prop to make settings link in sidebar visible again

## Issue(s) Resolved

Could not see Settings link in Sidebar even if you are superadmin.

## Test Plan

1. Login as a SuperAdmin
2. See the "Settings" link in the Sidebar
3. Click on it, and navigate to the tokens settings.

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
